<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigAudioProcessorsWidgetClass</class>
 <widget class="QWidget" name="ConfigAudioProcessorsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>535</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string notr="true">Audio processors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNoProcessorsMessage">
     <property name="text">
      <string>No processors available</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacked"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
